src/dataset-builder/CohortEditor.ts

```diff
@@ -54,23 +55,25 @@ const addCriteriaText = 'Add criteria';
 export const CriteriaView = (props: CriteriaViewProps) => {
   const { datasetId, criteria, deleteCriteria, updateCriteria } = props;
 
-  const [criteriaCount, setCriteriaCount] = useLoadedData<DatasetParticipantCountResponse>();
+  const [criteriaCount, setCriteriaCount] = useLoadedData<SnapshotBuilderCountResponse>();
 
   const updateCriteriaCount = useRef(
     _.debounce(250, (datasetId, criteria) => {
       setCriteriaCount(
         async () =>
           await DataRepo()
             .dataset(datasetId)
-            .getCounts({
-              cohorts: [
-                {
-                  // Create a "cohort" to get the count of participants for this criteria on its own.
-                  criteriaGroups: [{ criteria: [criteria], name: '', meetAll: true, mustMeet: true }],
-                  name: '',
-                },
-              ],
-            })
+            .getSnapshotBuilderCount(
+              convertDatasetParticipantCountRequest({
+                cohorts: [
+                  {
+                    // Create a "cohort" to get the count of participants for this criteria on its own.
+                    criteriaGroups: [{ criteria: [criteria], name: '', meetAll: true, mustMeet: true }],
+                    name: '',
+                  },
+                ],
+              })
```

Comment:
Similar to `convertDatasetAccessRequest`, I think this is no longer a `convert` operation but a `create` one, e.g., as a function that accepts a `cohorts` argument directly
```suggestion
              createDatasetParticipantCountRequest(
                [
                  {
                    // Create a "cohort" to get the count of participants for this criteria on its own.
                    criteriaGroups: [{ criteria: [criteria], name: '', meetAll: true, mustMeet: true }],
                    name: '',
                  },
                ])
```